TrafficCapture/dockerSolution/src/main/docker/migrationConsole/osiMigration.py

```diff
@@ -353,23 +398,26 @@ def create_pipeline_from_stage(osi_client, pipeline_name: str, pipeline_config_p
     pipeline_config = construct_pipeline_config(pipeline_config_file_path=pipeline_config_path, aws_region=region,
                                                 source_endpoint=source_endpoint, target_endpoint=target_endpoint,
                                                 source_auth_type='SIGV4', target_auth_type='SIGV4',
-                                                source_pipeline_role_arn=pipeline_role_arn,
-                                                target_pipeline_role_arn=pipeline_role_arn)
+                                                include_index_regex_list=include_index_regex_list,
+                                                pipeline_role_arn=pipeline_role_arn)
 
     if print_config_only:
         print(pipeline_config)
         exit(0)
 
     if print_command_only:
-        print(f"./osiMigration.py create-pipeline --source-endpoint={source_endpoint} "
-              f"--target-endpoint={target_endpoint} --aws-region={region} --subnet-ids={','.join(map(str,subnet_ids))} "
-              f"--security-group-ids={','.join(map(str,security_groups))} --source-auth-type='SIGV4' "
-              f"--target-auth-type='SIGV4' --source-pipeline-role-arn={pipeline_role_arn} "
-              f"--target-pipeline-role-arn={pipeline_role_arn} --tag=migration_deployment={solution_version}")
+        command_str = (f"./osiMigration.py create-pipeline --source-endpoint={source_endpoint} "
```

Comment:
Lets move each parameter on its own line to make this diffing these changes clearer in the future

Reply:
Have adjusted 👍